unlock-app/src/__tests__/paywall-builder/build.test.js

```diff
@@ -132,6 +143,7 @@ describe('buildPaywall', () => {
         expect(mockHide).toHaveBeenCalledTimes(1)
         expect(mockShow).toHaveBeenCalledTimes(1)
       })
+      it('calls redirect on redirect event', () => {})
```

Comment:
Ha! Did you forget to implement this one?

Reply:
no... it's.. a new post-modern test, the output is what you imagine it to be... yeah that's it...

Reply:
...and when implemented, I found a bug! Glad you caught this one